build.properties.dist

```diff
@@ -110,6 +110,12 @@ behat.yml.path = ${behat.dir}/behat.yml
 # The base URL to use in Behat tests.
 behat.base_url = http://localhost
 
+# The URL of the Behat webdriver host.
+behat.wd_host.url = http://localhost:8643/wd/hub
```

Comment:
That's by default in our VPS install. Maybe you could raise this to Ivan for the next VPS version?